libs/login/WordPressLoginFlow/src/main/res/layout/login_email_screen.xml

```diff
@@ -35,49 +35,82 @@
         tools:ignore="UnusedAttribute" >
     </org.wordpress.android.login.widgets.WPLoginInputRow>
 
+    <TextView
+        android:gravity="center_vertical"
+        android:layout_height="match_parent"
+        android:layout_marginBottom="@dimen/margin_medium"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:layout_width="wrap_content"
+        android:text="Alternatively:"
+        style="@style/LoginTheme.TextLabel" >
+    </TextView>
+
     <LinearLayout
-        android:id="@+id/login_google_layout"
+        android:id="@+id/login_google_button"
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/margin_extra_large"
-        android:layout_marginTop="@dimen/margin_extra_large"
-        android:layout_width="match_parent"
-        android:orientation="horizontal" >
+        android:layout_width="wrap_content"
+        android:orientation="horizontal"
+        android:paddingBottom="@dimen/margin_medium"
+        android:paddingEnd="@dimen/margin_medium"
+        android:paddingRight="@dimen/margin_medium"
+        android:paddingTop="@dimen/margin_medium"
+        tools:ignore="RtlSymmetry" >
+
+        <ImageView
+            android:importantForAccessibility="no"
+            android:layout_height="@dimen/google_button_icon_sz"
+            android:layout_marginEnd="@dimen/margin_medium"
+            android:layout_marginRight="@dimen/margin_medium"
+            android:layout_width="@dimen/google_button_icon_sz"
+            app:srcCompat="@drawable/ic_google_60dp" >
+        </ImageView>
 
         <TextView
             android:gravity="center_vertical"
             android:layout_height="match_parent"
             android:layout_width="wrap_content"
-            android:text="@string/login_google_button_prefix"
-            style="@style/LoginTheme.TextLabel" >
+            android:text="@string/login_google_button_suffix"
```

Comment:
Oh, I like that we simplified the text+icon+text into just icon+text!

Can we perhaps remove the `login_google_button_prefix` string resource now?

Reply:
Yes, we can.  I removed the string resource in aa1e0327d68e19796351ec425936e0cfc29b21d5.